tests/Endpoints/Users/ContractTest.php

```diff
@@ -69,4 +69,106 @@ public function testCreateContractFees(): void
          */
         self::assertInstanceOf(Ewallet::class, $contract->getEwallet());
     }
+
+    /**
+     * Test Contract fees getters.
+     *
+     * @return void
+     */
+    public function testGetters(): void
+    {
+        $user = new User(['id' => 'external-user-id']);
+        $ewallet = new Ewallet([]);
+        $contract = new Contract([
+            'action' => 'debit',
+            'currency' => 'AUD',
+            'ewallet' => $ewallet,
+            'group' => 'mastercard',
+            'fixed_fee' => '0.02',
+            'variable_rate' => '0.10',
+            'user' => $user
+        ]);
+
+        self::assertSame('debit', $contract->getAction());
+        self::assertSame('mastercard', $contract->getGroup());
+        self::assertSame('AUD', $contract->getCurrency());
+        self::assertSame($ewallet, $contract->getEwallet());
+        self::assertSame('0.02', $contract->getFixedFee());
+        self::assertSame('0.10', $contract->getVariableRate());
+        self::assertSame($user, $contract->getUser());
+    }
+
+    /**
+     * Test get endpoint is successful.
+     *
+     * @return void
+     */
+    public function testGetContractFees(): void
+    {
+        $apiManager = $this->createApiManager(
+            [
+                'created_at' => '2019-02-25T05:43:31Z',
+                'currency' => 'AUD',
+                'ewallet' => [
```

Comment:
Running the functional tests with xdebug for `FeeRetrieveControllerTest` in payments shows there's also a `type` and `action` included in the responses from Payments.